Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/3PZ0-Backup-Restore/backup-restore

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+set -x
+
+pod() {
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-openshift && bash Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/3PZ0-Backup-Restore/backup-restore node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"' '"'$RELEASE_TAG'"'
+}
+
+node() {
+source ~/.profile
+gittoken=$(echo "$github_token")
+
+job_id=$(echo $1)
+pipeline_id=$(echo $2)
+commit_id=$(echo $3)
+releaseTag=$(echo $4)
+case_id=3PZ0
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+
+bash Openshift-EE/utils/e2e-cr jobname:backup-restore jobphase:Waiting
+bash Openshift-EE/utils/e2e-cr jobname:backup-restore jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" openebs_version:"$releaseTag"
+
+################
+# LitmusBook 1 #
+################
+
+echo "*******Deploying Percona Application****"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=percona-deployment metadata="")
+echo $test_name
+
+cd litmus
+echo "Running the litmus test for Percona Deployment.."
+
+cp apps/percona/deployers/run_litmus_test.yml percona_backup_restore.yml
+
+sed -i -e 's/app: percona-deployment/app: deploy-percona-backup-restore/g' \
+-e 's/value: openebs-standard/value: openebs-cstor-disk/g' \
+-e 's/value: app-percona-ns/value: percona-backup/g' percona_backup_restore.yml
+cat percona_backup_restore.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:deploy-percona-backup-restore' job=percona_backup_restore.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:backup-restore $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id 3PZ0 3-functional "Create backup and restore and verify if the application is not impacted" Fail $pipeline_id "$current_time" $commit_id $gittoken
```

Comment:
"Create backup using velero and verify if it can be successfully restored" can be the proper job name.

Reply:
modifed